examples.md

```diff
@@ -853,6 +978,6 @@
     <li>Vestibulum auctor dapibus neque.</li>
   </ul>
 
-  <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Atque odio soluta fuga esse accusantium, voluptate, eos quod impedit! Quas cum saepe eaque quisquam culpa libero ipsum fuga necessitatibus tempore? Sequi!</p>
+  <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Atque odio soluta fuga esse accusantium, voluptate, eos quod impedit! Quas cum saepe eaque quisquam culpa libero ipsum fuga necessitatibus tempore? Sequi! Sequi!</p>
```

Comment:
I assume this is not a functional change?

Reply:
Sequi! An artifact of testing the Jekyll site compile that I was too lazy to remove.